api-gateway/src/main/resources/static/scripts/app.js

```diff
@@ -8,8 +8,8 @@ const whiteList = new Set([
 /* App Module */
 var petClinicApp = angular.module('petClinicApp', [
     'ui.router', 'layoutNav', 'layoutFooter', 'layoutWelcome', 'ownerList', 'ownerDetails', 'ownerForm', 'petForm'
-    , 'visits', 'vetList','vetForm','vetDetails', 'loginForm', 'rolesDetails', 'signupForm', 'billDetails', 'billHistory'
-    , 'verification' , 'adminPanel', 'visitList','billForm']);
```

Comment:
Could you clarify why visitList and billForm were removed?

Reply:
fixed